### DISCUSSION TITLE
node.js v14 hexo d Error The "mode" argument must be integer.

### DISCUSSION THREAD

**Discussion by HakunaMatata052** [hexojs/hexo#4858] [Question / Answer] [answered]

**Comment by yoshinorin** [comment, marked as answer]
@HakunaMatata052 
This issue is perhaps cased by https://github.com/hexojs/hexo/issues/4260.
It has been solved in hexo `5.0.0` Please update your site dependencies.

And please search & check existing issues or discussion before submit a new QA.
